test/emulation.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('Emulation', () => {
       function dispatchTouch() {
         let fulfill;
         const promise = new Promise((x) => (fulfill = x));
-        window.ontouchstart = function (e) {
+        window.ontouchstart = () => {
```

Comment:
But why though?

Reply:
the majority of the codebase + tests use arrow functions - this one stood out to me so I just swapped it out